src/sql/structure.sql

```diff
@@ -1222,6 +1222,7 @@ CREATE TABLE `users` (
   `valid_until` date NULL DEFAULT NULL,
   `entrypoint` tinyint UNSIGNED NOT NULL DEFAULT 0,
   `always_show_owned` TINYINT UNSIGNED NOT NULL DEFAULT 0,
+  `show_weekends` TINYINT UNSIGNED NOT NULL DEFAULT 1,
```

Comment:
what about having 0 as default? It would be a breaking change (not really an issue tbh), but what is the expected behavior for a professional tool? Probably to not see weekends, no? Hard to decide... :thinking: